mitti.js

```diff
@@ -8,19 +9,25 @@ function instance(system, id, config) {
 	instance_skel.apply(this, arguments);
 	self.actions(); // export actions
 	self.init_presets();
+	self.init_variables();
```

Comment:
This is double code. Put those in init and update.

Reply:
Done! Would you mind taking a look?

Reply:
should be fine now!